src/layouts/components/Header.tsx

```diff
@@ -237,3 +340,145 @@ function HeaderLink(props: {
+        text="Consortium"
+        pathname="/consortium/asset_breakdown"
+        testId={`${viewPort}.HeaderLink.Consortium`}
+      /> */}
+      <HeaderLink
+        className="flex justify-start border-b border-[#E5E5E5] dark:border-gray-700 px-4 p-1.5"
+        text="On-Chain Governance"
+        pathname="/on-chain-governance"
+        testId={`${viewPort}.HeaderLink.Consortium`}
+      />
+    </div>
+  );
+}
+
+function MoreDropdown(): JSX.Element {
+  const [isItemClicked, setIsItemClicked] = useState(false);
+  const router = useRouter();
+  useEffect(() => {
+    setIsItemClicked(
+      dropDownLinks.some((ddl) =>
+        router.pathname.includes(ddl.rootPathName.toLowerCase())
+      )
+    );
+  }, [router.pathname]);
+
+  return (
+    <Menu as="div" className="relative">
+      {({ open }) => (
+        <>
+          <Menu.Button
+            className={classNames(
+              "flex flex-row items-center mx-4 dark:hover:text-dark-50 cursor-pointer text-lg hover:text-primary-500",
+              {
+                "dark:text-dark-50 text-primary-500": isItemClicked,
+              }
+            )}
+          >
+            More
+            {open ? (
+              <IoChevronUp className="ml-2" size={20} />
+            ) : (
+              <IoChevronDown className="ml-2" size={20} />
+            )}
+          </Menu.Button>
+          <Transition
+            enter="transition duration-100 ease-out"
+            enterFrom="transform scale-95 opacity-0"
+            enterTo="transform scale-100 opacity-100"
+            leave="transition duration-75 ease-out"
+            leaveFrom="transform scale-100 opacity-100"
+            leaveTo="transform scale-95 opacity-0"
+          >
+            <Menu.Items className="absolute m-4 min-w-max flex flex-col divide-y bg-white border rounded-lg border-gray-200 dark:border-gray-700 dark:bg-gray-700">
+              {dropDownLinks.map((item, index) => {
+                return (
+                  <Menu.Item key={index}>
+                    <>
+                      <Link href={{ pathname: item.link }} passHref>
+                        <a
+                          className={classNames(
+                            "px-6 py-3.5 cursor-pointer text-sm border-gray-200 hover:text-primary-500 dark:hover:text-dark-50",
+                            {
+                              "dark:text-dark-50 text-primary-500":
+                                router.pathname.includes(item.rootPathName),
+                            }
+                          )}
+                        >
+                          {item.name}
+                        </a>
+                      </Link>
+                    </>
+                  </Menu.Item>
+                );
+              })}
+            </Menu.Items>
+          </Transition>
+        </>
+      )}
+    </Menu>
+  );
+}
+
+const dropDownLinks = [
```

Comment:
In future if we want to add new menu item the we have to update the entry at two places
1. at [link](https://github.com/WavesHQ/scan/blob/1a5994a882e99f84bf69687556eeeabf45b10dbf/src/layouts/components/Header.tsx#L196)
2. in dropDownLinks array [link](https://github.com/WavesHQ/scan/blob/1a5994a882e99f84bf69687556eeeabf45b10dbf/src/layouts/components/Header.tsx#L473)
I guess we can check a diff logic to address this

Reply:
For desktop tabs, new tabs will now be grouped under `More` dropdown there wouldnt be 2 points to update. Just at dropDownLinks array checks for width of screen is removed